doc/source/installation.rst

```diff
@@ -10,48 +10,42 @@ Follow these steps to successfully install Trident and its dependencies.
-<http://trident.readthedocs.io/en/latest>`_, and the `demeshening version
+There are two versions of Trident: the `stable version
+<http://trident.readthedocs.io/en/stable>`_, and the `developent version
+<http://trident.readthedocs.io/en/latest>`_.  The stable version is tried and
+tested, and it operates on the most recent stable version of yt.  The
+development version is actively being updated with new features, and it is
+also tied to the development version of yt, so occasionally unforseen bugs
+can crop up as these new features are added.
+
+If you are dealing with an SPH or Voronoi Tesselation code like Gadget, Gizmo,
+AREPO, Gasoline, or Changa, you will get best and fastest results from using the
+development versions of Trident and yt, since there have been a number of
+updates to these codes to better treat particle data natively (previously
+referred to as the demeshening.  For more details about the demeshening, see
+`this notebook
 <https://nbviewer.jupyter.org/url/trident-project.org/notebooks/trident_demesh_install.ipynb>`_.
-The stable version is tried and tested, and it normally operates on a stable
-version of yt.  The development version is actively being updated with new
-features, and it is also tied to the development version of yt, so occasionally
-unforseen bugs can crop up as these new features are added.  The demeshening
-version is currently in beta and active development and is used for better
-results on particle-based datasets.
-
-After Trident 1.2 was released on September 19, 2019, the demeshening version
-was merged with the main development branch. Now, there are only the stable
-version and the development version. 
-
-The installation steps are slightly different between the two versions,
-so pay attention in the steps below.  Don't worry if you want to change later,
-you can always switch between the two versions easily enough by following the
-directions in :ref:`uninstallation`.
-
-.. note::
-    The demeshening (development) version treats particle-based
-    datasets more natively.  The demeshening version will give faster and more 
-    accurate results with less memory overhead for particle-based datasets.  
-    For more information about the demeshening version, please see our
-    `demeshening notebook
-    <https://nbviewer.jupyter.org/url/trident-project.org/notebooks/trident_demesh_install.ipynb>`_. **Note, the installation instructions in the notebook should be ignored.**
-    Instead, follow the instructions for :ref:`install-dev`.
+
+The installation steps are slightly different between the two versions, in that
+the stable version of Trident requires a stable release of yt, whereas
+the development version of Trident requires the development version of yt, both
+built from source.  Don't worry if you want to change later, you can always
+switch between the two versions easily enough by following the directions in
+:ref:`uninstallation`.
 
 .. _step-1:
```

Comment:
Since yt is a hard dependency for trident, it should be possible to just to `pip install trident` to get stable releases of both codes.  It might be useful to add a suggestion to try that before installing yt and trident separately.